Source/cryptography/cryptography.h

```diff
@@ -21,3 +21,26 @@
 
 #include "ICryptography.h"
 #include "INetflixSecurity.h"
+
+#undef EXTERNAL
+#if defined(WIN32) || defined(_WINDOWS) || defined (__CYGWIN__) || defined(_WIN64)
+#ifdef CRYPTOGRAPHY_EXPORTS
+#define EXTERNAL __declspec(dllexport)
+#else
+#define EXTERNAL __declspec(dllimport)
+#pragma comment(lib, "cryptography.lib")
+#endif
+#else
+#define EXTERNAL __attribute__((visibility("default")))
+#endif
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+EXTERNAL void cryptography_dispose();
```

Comment:
Can you please explain why you added this? I see that there was a  WPEFramework::Implementation::CryptographyLink::Dispose(); in the test teardown, I guess that is not working anymore now? We would have expected that  WPEFramework::Implementation::CryptographyLink::Dispose(); would not have been needed at all in the test teardown but that releasing all refcounts to the interfaces acquired in the tests (they are IUknown tests so they do support reference counting) in the teardown would achieve the same effect.

Reply:
The test code was not compilable earlier. since this symbol is undefined from the test scope

Reply:
removed teardown interfaces, since the release has to be managed based on the object constructed and added release based on that